nifi-commons/nifi-property-protection-factory/pom.xml

```diff
@@ -21,6 +21,11 @@
         <version>1.16.0-SNAPSHOT</version>
     </parent>
     <artifactId>nifi-property-protection-factory</artifactId>
+    <properties>
+        <!-- We appear to not be able to use the same version of Jackson here because of a subtle bug related to it
+        in the Azure Identity lib-->
+        <jackson.version>2.12.4</jackson.version>
+    </properties>
```

Comment:
I noticed the following error on the previous build:
```
javax.xml.stream.FactoryConfigurationError: Provider for javax.xml.stream.XMLInputFactory cannot be found
```

Can the Azure library be upgraded, instead of hard-coding a particular Jackson version?

Reply:
I added `woodstox` to satisfy an equivalent issue in the Registry. I'll try that here.